orc8r/cloud/helm/orc8r/templates/obsidian.service.yaml

```diff
@@ -0,0 +1,31 @@
+{{/*
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+*/}}
+
+{{- include "orc8rlib.service" (list . "obsidian.service") -}}
+{{- define "obsidian.service" -}}
+metadata:
+  name: {{ .Release.Name }}-obsidian
+  labels:
+    {{- with .Values.obsidian.service.labels }}
+{{ toYaml . | indent 4}}
+    {{- end}}
+  {{- with .Values.obsidian.service.annotations }}
+  annotations:
+{{ toYaml . | indent 4}}
+  {{- end }}
+spec:
+  ports:
+    - name: grpc
+      port: 9180
+      targetPort: 9093
```

Comment:
doesn't this one need an http port for echo?

Reply:
Yeah good catch. Nginx still uses `orc8r-controller`, but will need this once we switch